commit/report.go

```diff
@@ -172,6 +178,40 @@ func (p *Plugin) ShouldTransmitAcceptedReport(
 	return true, nil
 }
 
+// cursingValidation will make one contract call to get rmn curse info.
+// If destination is cursed or some source chain is cursed it returns error.
+func cursingValidation(
+	ctx context.Context,
+	ccipReader ccipreader.CCIPReader,
+	merkleRoots []cciptypes.MerkleRootChain,
+) error {
+	if len(merkleRoots) == 0 {
+		return nil
```

Comment:
Maybe add a comment for this case, i.e we still want to transmit tokprices/gasprices if cursed